test/forge/metamorpho/TestMetamorphoSnippets.sol

```diff
@@ -119,13 +119,23 @@ contract TestIntegrationSnippets is IntegrationTest {
         assertEq(Id.unwrap(withdrawQueueList[1]), Id.unwrap(expectedWithdrawQueue[1]));
     }
 
-    function testCapMarket(MarketParams memory marketParams) public {
-        Id idMarket = marketParams.id();
+    function testTotalCapAsset(uint256 capMarket1, uint256 capMarket2, uint256 capMarket3) public {
+        capMarket1 = bound(capMarket1, MIN_TEST_ASSETS, MAX_TEST_ASSETS);
+        capMarket2 = bound(capMarket2, MIN_TEST_ASSETS, MAX_TEST_ASSETS);
+        capMarket3 = bound(capMarket3, MIN_TEST_ASSETS, MAX_TEST_ASSETS);
 
-        assertEq(vault.config(idMarket).cap, snippets.capMarket(address(vault), marketParams), "cap per market");
+        _setCap(allMarkets[0], capMarket1);
+        _setCap(allMarkets[1], capMarket2);
+        _setCap(allMarkets[2], capMarket3);
+
+        assertEq(
+            capMarket1 + capMarket2 + capMarket3,
+            snippets.totalCapAsset(address(vault), address(collateralToken)),
+            "cap per market"
```

Comment:
```suggestion
            "total collateral cap"
```

Reply:
though you could also remove it considering it's the only test of this test function